### PR TITLE
Enable add OAuth token to headers

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: client.jsx [hunk 1]
```diff
@@ -35,6 +35,10 @@ export default class Client {
         this.defaultHeaders['Accept-Language'] = locale;
     }
 
+    setOAuthToken(token) {
```
Comment (coreyhulen) [inline client.jsx]: I think there is `useHeaderToken()` which does the same thing?

Comment (enahum) [inline client.jsx]: @coreyhulen now?

**Comment by jwilander** [comment]
+1

**Comment by hmhealey** [comment]
+1 but deferring to Corey

**Comment by coreyhulen** [comment]
+1

**Pull request merged**